wiki/iNiS_games/de.md

```diff
@@ -0,0 +1,36 @@
+---
+stub: true
+tags:
+  - DS
+  - EBA
+  - Elite Beat Agents
+  - iNiS Corporation
+  - Nintendo
+  - Nintendo DS
+  - original game
+  - original games
+  - Osu! Tatakae! Ouendan!
+  - Osu Tatakae Ouendan
+  - Ouendan
+  - originales Spiel
+---
+
+# Spiele von iNiS
+
+*Siehe auch: [osu! (Begriffsabgrenzung)](/wiki/Disambiguation/osu!)*
+
+Der [Spielmodus](/wiki/Game_mode) [osu!](/wiki/Game_mode/osu!) und osu! selbst stammen ursprünglich von den Rhythmusspielen ab, die von **[iNiS](https://en.wikipedia.org/wiki/INiS_Corporation)** für den [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS) in den frühen 2000er Jahren entwickelt wurden. ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) hat osu! als einen Simulator und Level-Editor für die Spiele von iNiS gebaut und die frühesten Mitglieder der Community waren allgemein auch von denen Fans. Ältere [Beatmaps](/wiki/Beatmap) waren stark von dem Leveldesign von iNiS beeinflusst.
```

Comment:
```suggestion
Der [Spielmodus](/wiki/Game_mode) [osu!](/wiki/Game_mode/osu!) und osu! selbst stammen ursprünglich von den Rhythmusspielen ab, die von **[iNiS](https://en.wikipedia.org/wiki/INiS_Corporation)** für den [Nintendo DS](https://de.wikipedia.org/wiki/Nintendo_DS) in den frühen 2000er Jahren entwickelt wurden. ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) hat osu! als einen Simulator und Level-Editor für die Spiele von iNiS gebaut und die frühesten Mitglieder der Community waren allgemein auch von denen Fans. Ältere [Beatmaps](/wiki/Beatmap) waren stark von dem Leveldesign von iNiS beeinflusst.
```
might aswell link to the german wikipedia article if one exists

Reply:
I remember looking for a German article yesterday for a few links here. Must have forgotten to edit it in the article.